lightapi/auth.py

```
@@ -29,90 +30,130 @@ def authenticate(self, request: Request) -> bool:
         """
         return True
 
+    def get_auth_error_response(self, request: Request) -> JSONResponse:
```

Ruff (0.15.5)

[warning] Unused method argument: `request`

(ARG002)